js/templates/modals/editListing/coupon.html

```diff
@@ -0,0 +1,39 @@
+<div class="col4 simpleFlexCol">
```

Comment:
It's needed to make the fields line up if there are any errors:

![image](https://cloud.githubusercontent.com/assets/794517/22837276/badf4862-ef74-11e6-823e-a8d0065e4673.png)